crates/nu-command/src/conversions/into/filesize.rs

```diff
@@ -140,14 +145,28 @@ fn int_from_string(a_string: &str, span: Span) -> Result<i64, ShellError> {
     // Now that we know the locale, get the thousands separator and remove it
     // so strings like 1,123,456 can be parsed as 1123456
     let no_comma_string = a_string.replace(locale.separator(), "");
```

Comment:
```suggestion
    let no_comma_string = a_string.replace(locale.separator(), "").trim();
```

I think it would better to trim here, before checking if it starts with `"-"`.

Reply:
Updated